packages/as-transformer/src/transformer.ts

```diff
@@ -57,16 +56,55 @@ export class Transformer extends TransformVisitor {
     return super.visitCallExpression(node);
   }
 
-  private _updateSource(oldSource: Source, newContent: string, parser: Parser) {
+  /**
+   * Updates the given source file with the given new content content.
+   *
+   * @param oldSource - The old source file to be updates
+   * @param newContent - The new file content for the source file
+   * @param parser - The parser of the {@link afterParse} hook
+   *
+   * @returns The updated source.
+   */
+  private _updateSource(
+    oldSource: Source,
+    newContent: string,
+    parser: Parser,
+  ): Source {
     let newParser = new Parser(parser.diagnostics);
     newParser.parseFile(newContent!, oldSource.internalPath + '.ts', true);
 
     let newSource = newParser.sources.pop()!;
     utils.updateSource(this.program, newSource);
-    console.log('Updating source for: ' + oldSource.internalPath);
     return newSource;
```

Comment:
I'm returning the updated source in case any future transformer as to make such post 'visiting' source updates,
so that they work on the same source and don't overwrite each other. 

So yes, for the moment it doesn't serve any purpose since 'massaExport' is the only transformer with such feature.